armi/reactor/blockParameters.py

```diff
@@ -84,19 +84,25 @@ def getBlockParameterDefinitions():
         pb.defParam(
             "molesHmBOL",
             units=f"{units.MOLES}",
-            description="Total number of atoms of heavy metal at BOL assuming a full assembly",
+            description="Total number of atoms of heavy metal at BOL.",
+            location=ParamLocation.VOLUME_INTEGRATED,
```

Comment:
This works, but it's a little bit confusing to have these parameters defined within a context manager with `location=ParamLocation.AVERAGE`.

Reply:
Moved!